src/Application/Features/Participants/DTOs/ParticipantDto.cs

```diff
@@ -12,6 +12,8 @@ public class ParticipantDto
     public string? MiddleName { get; set; }
     public string? LastName { get; set; }
     public DateOnly? DateOfBirth { get; set; }
+    public DateTime? Due { get; set; }
+    public int? DueInDays { get; set; }
```

Comment:
```suggestion
    public DateTime? RiskDue { get; set; }
    public int? RiskDueInDays { get; set; }
```
This is fine, but is not completely obvious this is related to the 'risk' itself, as these properties exist in the context of a participant. I would suggest prefixing these properties with `Risk`.